ios/FluentUI.Demo/FluentUI.Demo/Resources/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -1,107 +1,116 @@
 {
   "images" : [
     {
+      "filename" : " 20@2x-1.png",
       "idiom" : "iphone",
-      "size" : "20x20",
-      "scale" : "2x"
+      "scale" : "2x",
+      "size" : "20x20"
     },
     {
+      "filename" : " 20@3x.png",
       "idiom" : "iphone",
-      "size" : "20x20",
-      "scale" : "3x"
+      "scale" : "3x",
+      "size" : "20x20"
     },
     {
-      "size" : "29x29",
+      "filename" : " 29@2x-1.png",
       "idiom" : "iphone",
-      "filename" : "Icon-29_2x.png",
-      "scale" : "2x"
+      "scale" : "2x",
+      "size" : "29x29"
     },
     {
-      "size" : "29x29",
+      "filename" : " 29@3x.png",
       "idiom" : "iphone",
-      "filename" : "Icon-29_3x.png",
-      "scale" : "3x"
+      "scale" : "3x",
+      "size" : "29x29"
     },
     {
-      "size" : "40x40",
+      "filename" : " 40@2x-1.png",
       "idiom" : "iphone",
-      "filename" : "Icon-40_2x.png",
-      "scale" : "2x"
+      "scale" : "2x",
+      "size" : "40x40"
     },
     {
-      "size" : "40x40",
+      "filename" : " 40@3x.png",
       "idiom" : "iphone",
-      "filename" : "Icon-40_3x.png",
-      "scale" : "3x"
+      "scale" : "3x",
+      "size" : "40x40"
     },
     {
-      "size" : "60x60",
+      "filename" : " 60@2x.png",
       "idiom" : "iphone",
-      "filename" : "Icon-60_2x.png",
-      "scale" : "2x"
+      "scale" : "2x",
+      "size" : "60x60"
     },
     {
-      "size" : "60x60",
+      "filename" : " 60@3x.png",
       "idiom" : "iphone",
-      "filename" : "Icon-60_3x.png",
-      "scale" : "3x"
+      "scale" : "3x",
+      "size" : "60x60"
     },
     {
+      "filename" : " 20.png",
       "idiom" : "ipad",
-      "size" : "20x20",
-      "scale" : "1x"
+      "scale" : "1x",
```

Comment:
Do we need the 1x images? Does anything use them?

Reply:
not sure. maybe not? but it was already in the appicon template so i just add them in any way.